extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -191,7 +191,13 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 				}
 			}
 		}
+	}
 
+	public async searchJupyterBooks(): Promise<void> {
+		if (this.currentBook.bookPath) {
```

Comment:
What if this.currentBook is undefined? Isn't that the case when searching from the command palette (which is a valid entry point here)?

Reply:
Thanks!